iModelJsNodeAddon/IModelJsNative.cpp

```diff
@@ -525,11 +525,37 @@ struct NativeECDb : BeObjectWrap<NativeECDb>, SQLiteOps {
             }
         }
     }
+
     Napi::Value SharedChannelEnabled(NapiInfoCR info) {
         RequireDbIsOpen(info);
         const auto isEnabled = !m_ecdb.Schemas().GetSharedChannel().GetInfo().IsEmpty();
         return Napi::Boolean::New(Env(), isEnabled);
     }
+
+    Napi::Value SharedChannelGetLocalInfo(NapiInfoCR info) {

```

Comment:
again, can we please not call all of this "channels"? We already use that term for access control to models.

Reply:
I am renaming everything removing the term shared/channel and use schema sync.